### PR TITLE
External tool setup for receiving notifications

### PR DESCRIPTION
**Pull request by nightfurys**

### CODE REVIEW

File: content/docs/overview/notifications/_index.md [hunk 1]
```diff
@@ -127,6 +136,6 @@ Selector configurations for notifying a few interesting events
 
 #### Enterprise UI
 
-To learn more about configuring Notifications service in the Enterprise UI go to [Notifications]({{< ref "/docs/using/ui_usage/reports" >}})             
+[Click here] to learn more about configuring notifications in the Enterprise UI
```
Comment (zhill) [inline content/docs/overview/notifications/_index.md]: Should this have a link?

File: content/docs/overview/notifications/github.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+1. The following GitHub account and repository related information is required
+    * Username of the account for creating issues
+    * Token with repository access. [Follow instructions](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line#creating-a-token) to create a new Access Token in the account used for creating issues  
+    * Name of the repository where the issues are created
+    * Owner of the repository 
+    * (Optional) GitHub API URL, defaults to https://api.github.com if not specified
+    * (Optional) milestone assigned to the issue
+    * (Optional) one or more labels assigned to the issue
+    * (Optional) one or more GitHub users assigned to the issue 
+2. Create a GitHub endpoint configuration in the Notifications service either via Enterprise UI or the API directly
```
Comment (zhill) [inline content/docs/overview/notifications/github.md]: Probably link here to the doc to do that setup

File: content/docs/overview/notifications/jira.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    * Username of the account for creating issues
+    * API token or password depending on the Jira project
+        * For Jira Cloud projects an API token is required. [Follow instructions](https://confluence.atlassian.com/cloud/api-tokens-938839638.html) to create a new API token for the account creating issues
+        * For Jira Self-managed projects password of the account creating issues is required 
+    * Project key, same as the prefix in the issue identifier. For instance, issue `TEST-1` has the project key `TEST`
+    * Type of the issue created such as `Bug`, `Task`, `Story`, etc. 
+    * (Optional) priority assigned to the issue such as `Low`, `Medium`, `High`, etc. 
+    * (Optional) one or more labels to be assigned to the issue
+    * (Optional) Jira user to be assigned to the issue 
+2. Create a Jira endpoint configuration in the Notifications service either via Enterprise UI or the API directly
```
Comment (zhill) [inline content/docs/overview/notifications/jira.md]: Probably link here to the doc to do that setup

File: content/docs/overview/notifications/slack.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+Notifications to Slack are in the form of messages to a channel
+
+### Requirements
+
+To receive notifications
+
+1. An Incoming Webhook URL is required. [Follow instructions](https://api.slack.com/messaging/webhooks) to create one
+2. Copy the Incoming Webhook URL
+3. Create a Slack endpoint configuration in the Notifications service either via Enterprise UI or the API directly
```
Comment (zhill) [inline content/docs/overview/notifications/slack.md]: Probably link here to the doc to do that setup

File: content/docs/overview/notifications/smtp.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+1. The following SMTP server information is required
+     * Host name/address
+     * Port number
+     * From address
+     * To address
+     * (Optional) username
+     * (Optional) password
+     * (Optional) use tls to encrypt connection
+2. Create an SMTP endpoint configuration in the Notifications service either via Enterprise UI or the API directly
```
Comment (zhill) [inline content/docs/overview/notifications/smtp.md]: Probably link here to the doc to do that setup

File: content/docs/overview/notifications/teams.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+Notifications to Microsoft Teams are in the form of messages to a channel
+
+### Requirements
+
+To receive notifications
+
+1. An incoming webhook URL is required. [Follow instructions](https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook#add-an-incoming-webhook-to-a-teams-channel) to create one
+2. Copy the incoming webhook URL
+3. Create a Microsoft Teams endpoint configuration in the Notifications service either via Enterprise UI or the API directly
```
Comment (zhill) [inline content/docs/overview/notifications/teams.md]: Probably link here to the doc to do that setup

**Comment by nightfurys** [comment]
@zhill, @katgurdak is working on the UI docs. I wanted to bring all the commits - ui and backend pages with links to each other's sections, through one PR initially. But I went ahead with my commits since the enterprise release has been out a few weeks and it seemed better to have some content rather than block on the perfect links. We can add the links after her content is committed

**Comment by nightfurys** [comment]
Closing this PR as #95 contains the same commits

**Pull request closed without merging**